scene/main/http_request.cpp

```diff
@@ -96,14 +96,16 @@ Error HTTPRequest::_parse_url(const String& p_url) {
 	return OK;
 }
 
-Error HTTPRequest::request(const String& p_url, const Vector<String>& p_custom_headers, bool p_ssl_validate_domain) {
+Error HTTPRequest::request(const String& p_url, const Vector<String>& p_custom_headers, bool p_ssl_validate_domain, HTTPClient::Method p_method, const String& p_request_data) {
 
 	ERR_FAIL_COND_V(!is_inside_tree(),ERR_UNCONFIGURED);
 	if ( requesting ) {
 		ERR_EXPLAIN("HTTPRequest is processing a request. Wait for completion or cancel it before attempting a new one.");
 		ERR_FAIL_V(ERR_BUSY);
 	}
 
+  method=p_method;
```

Comment:
This and other lines you added are indented with two spaces. Please use Tab as indent like the rest of the code.